client/src/main/java/io/pravega/client/segment/impl/SegmentInputStreamImpl.java

```diff
@@ -33,7 +33,9 @@
 @Slf4j
 @ToString
 class SegmentInputStreamImpl implements SegmentInputStream {
+    static final int MIN_BUFFER_SIZE = 1024;
```

Comment:
Could you explain why these new constants are needed?

Reply:
These are just sanity bounds, because we have made the buffer size configurable.